### DISCUSSION TITLE
What does 'sh' do?

### DISCUSSION THREAD

**Discussion by ceanwang** [OpenRadioss/OpenRadioss#426] [Q&A] [answered]
Hi,
In CMakeLists.txt, there is below code:
`#include always rebuild 

add_custom_command(    

         OUTPUT 

            ${Build_includes_directory}/__foo.inc 

            ${Build_includes_directory}/build_info.inc

         COMMAND 

            sh ../../Compiling_tools/script/or_compile_info_cm.sh linux64 0 e 0 linux64  > ${Build_includes_directory}/build_info.inc

         )`

Does the 'sh' here just run this 'or_compile_info_cm.sh' script to create a file named build_info.inc? 
What is inside build_info.inc? I want to simulate the action under Windows.
Cheers,
Cean

**Comment by luzpaz** [comment]
Linking to #420  
Edit: This is not an answer

**Comment by OlivierWienholtz** [comment, marked as answer]
Hello,
sh is the shell interpreter. Today it invokes the bash Shell to run the script...

**Comment by OlivierWienholtz** [comment]
Hello, 
Did not understand your Question.
When cmake was crezted,  it was tested on various systems, cmake versions etc...
Had some issues where default interpreter is different (like Ubundu it is dash instead of bash).

As it is working as expected on many different platforms, it is safe to keep it like this.

**Reply by ceanwang** [reply]
I am doing it under Windows. There is no sh command under Windows. I want to know what is write into build_info.inc by this command:
`
Compiling_tools/script/or_compile_info_cm.sh linux64 0 e 0 linux64  > ${Build_includes_directory}/build_info.inc
`

**Comment by OlivierWienholtz** [comment]
Hello, 
I checked this, I think now it can be removed. 

Just to let you know, We are working on a solution for Windows building.
I'd be interesting to know what you are currently trying out.

Many thanks,

**Reply by ceanwang** [reply]
How to remove it?

Here is a screenshot what I got:

![openradioss1](https://user-images.githubusercontent.com/4224442/190372527-0e5125d3-dd4a-44b5-9ba4-7168c6f36abe.png)

**Comment by OlivierWienholtz** [comment]
Thank You !

I see now.

**Reply by ceanwang** [reply]
Could you share how to remove the sh command?

**Comment by OlivierWienholtz** [comment]
Hello,
We are actively working on Windows Porting. 
There are several libraries missing which will not enable Windows.

Pls wait until we release the Windows Porting.
